keps/sig-contributor-experience/0005-community-site.md

```diff
@@ -0,0 +1,104 @@
+---
+kep-number: 5
+title: Contributor Site
+authors:
+  - "@jbeda"
+owning-sig: sig-contributor-experience
+participating-sigs:
+  - sig-architecture
+reviewers:
+  - TBD
+approvers:
+  - TBD
+editor: TBD
+creation-date: "2018-02-19"
+last-updated: "2018-02-19"
+status: provisional
+---
+
+# Contributor Site
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+    * [User Stories [optional]](#user-stories-optional)
+      * [Story 1](#story-1)
+      * [Story 2](#story-2)
+    * [Implementation Details/Notes/Constraints [optional]](#implementation-detailsnotesconstraints-optional)
+    * [Risks and Mitigations](#risks-and-mitigations)
+* [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+* [Drawbacks [optional]](#drawbacks-optional)
+* [Alternatives [optional]](#alternatives-optional)
+
+## Summary
+
+We need a way to organize and publish information targeted at contributors.
+In order to continue to scale the Kubernetes community we need a convenient, scalable and findable way to publish information.
+
+## Motivation
+
+While the current kubernetes.io site is great for end users, it isn't often used by or aimed at project contributors.
+Instead, most contributors look at documentation in markdown files that are spread throughout a wide set of repos and orgs.
+It is difficult for users to find this documentation.
```

Comment:
It would be worth mentioning that standalone website will provide more visibility into the community process. Right now, as a potential new contributor or curious user, you have to know what to look for, what KEPs are etc. We could have that have an index of e.g. in-flight KEPs by SIG, which I think would be very helpful.
Also an index of community events, meetup groups would be very helpful.